stable_baselines3/common/policies.py

```diff
@@ -443,8 +443,17 @@ def __init__(
         self.activation_fn = activation_fn
         self.ortho_init = ortho_init
 
+        self.share_features_extractor = self.features_extractor_kwargs.get("shared", True)
```

Comment:
I would rather add a parameter to the class, even though it means updating all classes.

Reply:
Alright! I had put it in the `features_extractor_kwargs` in order not to introduce too many modifications (and also because it's an argument related to the features extractor), but I'll put it in the class as you said 👌